api/src/opentrons/protocol_api/labware.py

```diff
@@ -644,13 +640,13 @@ def wells(self, *args: Union[str, int]) -> List[Well]:
     @requires_version(2, 0)
     def wells_by_name(self) -> Dict[str, Well]:
         """
-        Accessor function used to create a look-up table of Wells by name.
+        Accessor function used to create a look-up table of wells by name.
 
         With indexing one can treat it as a typical python
-        dictionary whose keys are well names. To access well A1, for example,
-        write: labware.wells_by_name()['A1']
+        dictionary whose keys are well names. To access well A1 for example,
+        write ``labware.wells_by_name()['A1']``.
```

Comment:
Suggesting same change as for `wells()` above.
```suggestion
        dictionary whose keys are well names. For example, access well A1
        with ``labware.wells_by_name()['A1']``.
```

Reply:
Will fix in separate push.